fortuna/src/backend/routes/api/auth.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const auth = require('../middleware/auth');
+const jwt = require('jsonwebtoken');
+const { check, validationResult } = require('express-validator');
+const config = require('config');
+const bcrypt = require('bcrypt');
+
+
+const User = require('../../../models/userModel');
+
+
+router.get('/', auth, async (req, res) => {
```

Comment:
What types are req and res expected to be? specifying this with flow would be helpful I think

Reply:
> What types are req and res expected to be? specifying this with flow would be helpful I think

I would think it means request and response returned from the async method.